ktor-server/ktor-server-core/jvmAndNix/src/io/ktor/server/application/CreatePluginUtils.kt

```diff
@@ -140,27 +142,33 @@ public fun createApplicationPlugin(
  **/
 public fun createRouteScopedPlugin(
     name: String,
-    body: PluginBuilder<Unit>.() -> Unit
+    body: ApplicationPluginBuilder<Unit>.() -> Unit
```

Comment:
Should it be `RouteScopedPluginBuilder`? Or, if there is only ine class, why add `Application` to it?

Reply:
We didn't ever have a specific plugin builder. It's just a consequence of renaming `PluginBuilderBase` -> `PluginBuilder` and `PluginBuilder` to `ApplicationPluginBuilder`. I can introduce type alias `RoutingPluginBuilder = ApplicationPluginBuilder`.

Reply:
Why not just `PluginBuilder` then? We have 3 entities: `Plugin`, `ApplicationPlugin` and `RouteScopedPlugin`. I think `ApplicationPluginBuilder ` creates a confustion here.